.github/pull_request_template.md

```diff
@@ -14,3 +14,4 @@ preview URL:
 <!-- Check completed items of strikethrough irrelevant items (using ~~text~~) -->
 - [ ] All necessary documentation has been adapted.
 - [ ] The implemented feature is covered by an appropriate test.
```

Comment:
Maybe add "automated" before the "test" here, to distinguish from manual tests :)